packages/wallets/solflare/package.json

```diff
@@ -36,7 +36,7 @@
     },
     "dependencies": {
         "@solana/wallet-adapter-base": "workspace:^",
-        "@solflare-wallet/sdk": "^1.2.1"
+        "@solflare-wallet/sdk": "^1.3.0"
```

Comment:
Will need to be updated with a type fix, see below.

Reply:
@jordansexton Can you tell me which type is missing? The source is in TS so the declarations are generated automatically and we have this in both `lib/cjs/index.d.ts` and `lib/esm/index.d.ts`:

```
signAndSendTransaction(transaction: TransactionOrVersionedTransaction, options?: SendOptions): Promise<string>;
```

Reply:
Hmm, you must be right, I went back into the project branch and the method was there. Must have been a package installation problem.